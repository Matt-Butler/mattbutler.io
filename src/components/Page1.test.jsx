import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Page1 from './Page1';

it('matches snapshot', () => {
  const wrapper = shallow(<Page1 />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
