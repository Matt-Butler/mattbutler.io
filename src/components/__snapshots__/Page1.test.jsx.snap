// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  Page1
</div>
`;
